docker-app/qfieldcloud/core/models.py

```diff
@@ -1159,6 +1098,28 @@ def clean(self) -> None:
         return super().clean()
 
 
+class ProjectPermissionsView(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.DO_NOTHING,
+        related_name="related_projects",
```

Comment:
I'd use `projects_permissions` as related name (otherwise, `user1.related_projects.all()` is confusing, as you'd expect it to be a list of projects)

Reply:
What if we rename `related_name="project_roles"`?